common/src/commonMain/kotlin/io/xorum/codeforceswatcher/features/users/redux/requests/UsersRequests.kt

```diff
@@ -83,7 +83,7 @@ class UsersRequests {
 
         override suspend fun execute() {
             when (val response = backendRepository.fetchUsers(handle, isAllRatingChangesNeeded = false)) {
-                is Response.Success -> response.result.firstOrNull()?.let { user -> addUser(user) }
+                is Response.Success -> response.result.firstOrNull()?.let { user -> addUser(user) } ?: store.dispatch(Failure(null.toMessage()))
```

Comment:
I think it's impossible to reach such situation.

Reply:
when user have russian name

Reply:
ok